src/frontend/Komponenter/Behandling/Brev/Brev.tsx

```diff
@@ -43,10 +58,13 @@ const Brev: React.FC<Props> = ({ behandlingId }) => {
         useBehandling();
     const [kanSendesTilBeslutter, settKanSendesTilBeslutter] = useState<boolean>(false);
     const { hentVedtak, vedtak } = useHentVedtak(behandlingId);
+    const oppgaverForOpprettelseState = useHentOppgaverForOpprettelse();
+    const { hentOppgaverForOpprettelse } = oppgaverForOpprettelseState;
 
     useEffect(() => {
         hentVedtak();
-    }, [hentVedtak]);
+        hentOppgaverForOpprettelse(behandlingId);
```

Comment:
Kan hooken ha ansvar for å kalle backend for å hente oppgaver som kan og evt. skal opprettes? Legge til en useEffect i hooken